Makefile

```diff
@@ -163,6 +163,14 @@ _opt_pylint:
 	@command -v flake8 && { $(MAKE) pylint; exit $$?; } \
 		|| echo "SKIP: pylint (flake8 not found)"
 
+current_branch=$(shell git branch --show-current)
+ancestor_commit=$(shell git merge-base master $(current_branch))
+commits_on_current_pr=$(shell git rev-list $(ancestor_commit)..$(current_branch))
```

Comment:
this deserves a separate script, `scripts/lintcommits.lua`

Reply:
I'm getting `E5107: Error loading lua [string ":lua"]:1: '=' expected near '<eof>'`. I've tried it with other lua files as well, same error.

Reply:
sorry i meant `:luafile` . 

    nvim +"luafile scripts/lintcommit.lua"